### DISCUSSION TITLE
Fob Building Disappeared

### DISCUSSION THREAD

**Discussion by Anderanged** [KillahPotatoes/KP-Liberation#901] [Q&A]
Two fobs' HQs models have vanished. It seems like they're deleted, but their added actions (redeploy, halo, arsenal, and build) still appear in the scroll menu. I cannot repackage them. I assume it was something to do with saving screwing up as a side effect of my incompetence, but with that now seemingly resolved I just want to remove the markers and actions from those sections of the map. 

I attempted removing them from GRLIB_all_fobs, but that seems to have done nothing (as it should have, I suppose). Is there a more straightforward way to accomplish this removal besides taking do_repackage_fob.sqf and altering it for my purposes? 

Cheers

**Comment by Red-Dog-Actual** [comment]
Having the same issue on our current mission. Everything built at the FOB vanished on the third server restart, and has continued to do so ever since.
